### ISSUE TITLE
Opsdroid does not exit with ctrl-c

### ISSUE THREAD

**Issue by chillipeper** [opsdroid/opsdroid#1704]
# Description
Opsdroid is not exiting with ctrl-c. When starting Opsdroid locally, I was able to exit the bot with the ctrl-c escape key.  Now it only hangs, and I have to kill the process manually to exit,

## Steps to Reproduce
- Clone the repo and install the latest branch of Opsdroid
- execute `opsdroid start`

I have tested this without a configuration file, so it uses the default one, and also is happening when trying to connect with the slack connector.

## Expected Functionality
ctrl-c signal should work when executed.

## Experienced Functionality
- ctl-c signal is not working and opsdroid just hangs there. I has to be manually killed

## Versions
- **Opsdroid version:** v0.19.0+84.ga99dfb6
- **Python version:** Tested in 3.7.7 and 3.8.5 same issue on both
- **OS/Docker version:** Ubuntu 20.04

## Additional Details
Any other details you wish to include such as screenshots, console messages, etc.

```

--> opsdroid start
INFO opsdroid.logging: ========================================
INFO opsdroid.logging: Started opsdroid v0.19.0+84.ga99dfb6.
INFO opsdroid: ========================================
INFO opsdroid: You can customise your opsdroid by modifying your configuration.yaml.
INFO opsdroid: Read more at: http://opsdroid.readthedocs.io/#configuration
INFO opsdroid: Watch the Get Started Videos at: http://bit.ly/2fnC0Fh
INFO opsdroid: Install Opsdroid Desktop at: 
https://github.com/opsdroid/opsdroid-desktop/releases
INFO opsdroid: ========================================
WARNING opsdroid.loader: No databases in configuration. This will cause skills which store things in memory to lose data when opsdroid is restarted.


INFO opsdroid.core: Opsdroid is now running, press ctrl+c to exit.
INFO opsdroid.web: Started web server on http://0.0.0.0:8080
^C^Cc^C^C^C^C^C^C
```

**Comment by jacobtomlinson** [comment]
cc @cognifloyd. I think this may be due to recent signalling work.

**Comment by cognifloyd** [comment]
Should be fixed with #1708
